code/codescanner.py

```diff
@@ -0,0 +1,37 @@
+from codescanner_analysis import CodescannerAnalysisData
+
+from analysis.PluginBase import AnalysisBasePlugin
+from objects.file import FileObject
+from plugins.mime_blacklists import MIME_BLACKLIST_COMPRESSED, MIME_BLACKLIST_NON_EXECUTABLE
+
+
+class AnalysisPlugin(AnalysisBasePlugin):
+
+    NAME = 'codescanner'
+    DESCRIPTION = 'scan unknown binaries for executable code'
+    VERSION = '1.0.0'
+    DEPENDENCIES = ['file_type']
+    MIME_BLACKLIST = MIME_BLACKLIST_NON_EXECUTABLE + MIME_BLACKLIST_COMPRESSED
```

Comment:
`MIME_BLACKLIST_NON_EXECUTABLE`.

I'm wondering if running the tool on already identified executable files might still be reasonable. The tool provides a lot of interesting information and the plots might be a valuable contribution for further in-depth analysis. Think about a packed ELF files, you could probably see that pretty fast using codescanner plots.

What do you think? Lets talk! :-)

Reply:
We talked about it. The blacklist stays as-is.